src/model/names.mli

```diff
@@ -4,15 +4,39 @@ module type Name = sig
 
     val to_string : t -> string
 
+    val to_string_unsafe : t -> string
```

Comment:
It's not entirely clear here what's unsafe about this function -- conventionally, that would be some lack of bounds checking for string functions, but here it's due to the abstraction leak of the internal name representation. Perhaps it's worth clarifying in the docstring for the function what the impact of using this function might be.

Reply:
Good point - I'll clarify that.